Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -928,4 +929,13 @@
         </Target>
     </ECRelationshipClass>
 
+    <ECRelationshipClass typeName="AuthoringElementDrivesPlanElement" modifier="None" strength="referencing">
+        <BaseClass>bis:ElementDrivesElement</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="creates" polymorphic="true">
+            <Class class="SpatialLocationElement"/>
```

Comment:
For consistency add alias - `bis:SpatialLocationElement`. I think we always add alias when referencing other Schemas.